lua/plenary/path.lua

```diff
@@ -19,6 +19,24 @@ local S_IF = {
 local path = {}
 path.home = vim.loop.os_homedir()
 
+path.root = (function()
+    if jit then
+        local os = string.lower(jit.os)
+        if os == 'linux' or os == 'osx' or os == 'bsd' then
+            return '/'
+        else
+            return 'C:\\'
```

Comment:
Is this really correct for windows. Isn't there some sort of label system. So `F:\\` could also be root of a disk?
Just thinking out load i don't know anything about windows :laughing: